### PR TITLE
[DEVMGR] Update PT translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[DEVMGR] Update PT translation

Readjust objects size to fit all text in Driver tab

[NETID] Update PT translation

### CODE REVIEW

**Comment by zecarlos1957** [comment]
ReactOS

**Comment by zecarlos1957** [comment]
Need readjust objects size to fit all text in Driver tab

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 1]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
     IDS_DEV_NOT_CONFIGURED "Este dispositivo não está correctamente configurado."
-    IDS_DEV_DEVLOADER_FAILED "O ReactOS não pôde carregar o driver para este dispositivo porque o computador está a reportar dois tipos de barramento %1."
-    IDS_DEV_DEVLOADER_FAILED2 "O(s) carregadore(es) do dispositivo %1 não pôde/puderam carregar o driver do dispositivo para este dispositivo."
-    IDS_DEV_OUT_OF_MEMORY "O driver para este dispositivo pode estar danificado, ou seu sistema com pouca memória ou insuficiência de outros recursos."
-    IDS_DEV_ENTRY_IS_WRONG_TYPE "Este dispositivo não está funcionando correctamente porque um de seus drivers pode estar danificado, ou seu registo pode estar danificado."
-    IDS_DEV_LACKED_ARBITRATOR "O driver para este dispositivo requisitou um recurso que o ReactOS não sabe como manusear."
+    IDS_DEV_DEVLOADER_FAILED "O ReactOS não pôde carregar o controlador para este dispositivo porque o computador está a reportar dois tipos de barramento %1."
+    IDS_DEV_DEVLOADER_FAILED2 "O(s) carregadore(es) do dispositivo %1 não pôde/puderam carregar o controlador  para este dispositivo."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:44]:
```suggestion
    IDS_DEV_DEVLOADER_FAILED2 "O(s) carregadore(es) do dispositivo %1 não pôde/puderam carregar o controlador para este dispositivo."
```

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 2]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
-    IDS_DEV_INVALID_DATA2 "Este dispositivo não está a funcionar correctamente porque a BIOS do dispositivo está a reportar os recursos para este dispositivo incorretamente."
-    IDS_DEV_FAILED_START "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os drivers instalados."
+    IDS_DEV_FAILED_FILTER "Os controladores para este dispositivo precisam ser reinstalados."
+    IDS_DEV_DEVLOADER_NOT_FOUND "Este dispositivo não está a funcionar correctamente porque o ReactOS não pôde carregar o arquivo %1 que carrega os controladores para o dispositivo."
+    IDS_DEV_DEVLOADER_NOT_FOUND2 "Este dispositivo não está a funcionar correctamente porque o arquivo %1 que carrega os controladores para este dispositivo está danificado."
+    IDS_DEV_DEVLOADER_NOT_FOUND3 "Falha no dispositivo: tente mudar o controlador para este dispositivo. Se isto não funcionar, consulte a documentação do seu hardware."
+    IDS_DEV_INVALID_DATA "Este dispositivo não está a funcionar correctamente porque a BIOS do seu computador está a reportar os recursos para este dispositivo incorrectamente."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:53]: "incorrectamente" looks spanish? Isn't it "incorretamente" instead in portuguese?
also, "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 3]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
-    IDS_DEV_FAILED_DRIVER_ENTRY "O ReactOS não pode inicializar o driver do dispositivo para este hardware."
-    IDS_DEV_DRIVER_FAILED_PRIOR_UNLOAD "O ReactOS não pode carregar o driver do dispositivo para este hardware porque uma instância anterior do driver do dispositivo ainda está na memória."
-    IDS_DEV_DRIVER_FAILED_LOAD "O ReactOS não pode carregar o driver do dispositivo para este hardware. O driver pode estar corrompido ou ausente."
+    IDS_DEV_FAILED_DRIVER_ENTRY "O ReactOS não pode inicializar o controlador do dispositivo para este hardware."
+    IDS_DEV_DRIVER_FAILED_PRIOR_UNLOAD "O ReactOS não pode carregar o controlador do dispositivo para este hardware porque uma instância anterior do controlador do dispositivo ainda está na memória."
+    IDS_DEV_DRIVER_FAILED_LOAD "O ReactOS não pode carregar o controlador do dispositivo para este hardware. O controlador pode estar corrompido ou ausente."
     IDS_DEV_DRIVER_SERVICE_KEY_INVALID "O ReactOS não pode acessar este hardware porque suas informações sobre a chave do serviço não estão no Registo ou estão registadas incorrectamente."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:87]: "incorretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 4]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
-    IDS_DEV_FAILED_START "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os drivers instalados."
+    IDS_DEV_FAILED_FILTER "Os controladores para este dispositivo precisam ser reinstalados."
+    IDS_DEV_DEVLOADER_NOT_FOUND "Este dispositivo não está a funcionar correctamente porque o ReactOS não pôde carregar o arquivo %1 que carrega os controladores para o dispositivo."
+    IDS_DEV_DEVLOADER_NOT_FOUND2 "Este dispositivo não está a funcionar correctamente porque o arquivo %1 que carrega os controladores para este dispositivo está danificado."
+    IDS_DEV_DEVLOADER_NOT_FOUND3 "Falha no dispositivo: tente mudar o controlador para este dispositivo. Se isto não funcionar, consulte a documentação do seu hardware."
+    IDS_DEV_INVALID_DATA "Este dispositivo não está a funcionar correctamente porque a BIOS do seu computador está a reportar os recursos para este dispositivo incorrectamente."
+    IDS_DEV_INVALID_DATA2 "Este dispositivo não está a funcionar correctamente porque a BIOS do dispositivo está a reportar os recursos para este dispositivo incorrectamente."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:54]: "corretamente"  and  "incorretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 5]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
 BEGIN
     IDS_DEV_NO_PROBLEM "Este dispositivo está a funcionar correctamente."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:41]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 6]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
     IDS_DEV_NOT_CONFIGURED "Este dispositivo não está correctamente configurado."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:42]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 7]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
-    IDS_DEV_OUT_OF_MEMORY "O driver para este dispositivo pode estar danificado, ou seu sistema com pouca memória ou insuficiência de outros recursos."
-    IDS_DEV_ENTRY_IS_WRONG_TYPE "Este dispositivo não está funcionando correctamente porque um de seus drivers pode estar danificado, ou seu registo pode estar danificado."
-    IDS_DEV_LACKED_ARBITRATOR "O driver para este dispositivo requisitou um recurso que o ReactOS não sabe como manusear."
+    IDS_DEV_DEVLOADER_FAILED "O ReactOS não pôde carregar o controlador para este dispositivo porque o computador está a reportar dois tipos de barramento %1."
+    IDS_DEV_DEVLOADER_FAILED2 "O(s) carregadore(es) do dispositivo %1 não pôde/puderam carregar o controlador  para este dispositivo."
+    IDS_DEV_OUT_OF_MEMORY "O controlador para este dispositivo pode estar danificado, ou o sistema com pouca memória ou insuficiência de outros recursos."
+    IDS_DEV_ENTRY_IS_WRONG_TYPE "Este dispositivo não está a funcionar correctamente porque um de seus controladores pode estar danificado, ou o registo pode estar danificado."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:46]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 8]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
-    IDS_DEV_DEVLOADER_NOT_FOUND2 "Este dispositivo não está a funcionar correctamente porque o arquivo %1 que carrega os drivers para este dispositivo está danificado."
-    IDS_DEV_DEVLOADER_NOT_FOUND3 "Falha no dispositivo: tente mudar o driver para este dispositivo. Se isto não funcionar, consulte a documentação do seu hardware."
-    IDS_DEV_INVALID_DATA "Este dispositivo não está a funcionar correctamente porque a BIOS do seu computador está a reportar os recursos para este dispositivo incorretamente."
-    IDS_DEV_INVALID_DATA2 "Este dispositivo não está a funcionar correctamente porque a BIOS do dispositivo está a reportar os recursos para este dispositivo incorretamente."
-    IDS_DEV_FAILED_START "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os drivers instalados."
+    IDS_DEV_FAILED_FILTER "Os controladores para este dispositivo precisam ser reinstalados."
+    IDS_DEV_DEVLOADER_NOT_FOUND "Este dispositivo não está a funcionar correctamente porque o ReactOS não pôde carregar o arquivo %1 que carrega os controladores para o dispositivo."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:50]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 9]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
+    IDS_DEV_DEVLOADER_NOT_FOUND2 "Este dispositivo não está a funcionar correctamente porque o arquivo %1 que carrega os controladores para este dispositivo está danificado."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:51]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 10]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
+    IDS_DEV_DEVLOADER_NOT_FOUND3 "Falha no dispositivo: tente mudar o controlador para este dispositivo. Se isto não funcionar, consulte a documentação do seu hardware."
+    IDS_DEV_INVALID_DATA "Este dispositivo não está a funcionar correctamente porque a BIOS do seu computador está a reportar os recursos para este dispositivo incorrectamente."
+    IDS_DEV_INVALID_DATA2 "Este dispositivo não está a funcionar correctamente porque a BIOS do dispositivo está a reportar os recursos para este dispositivo incorrectamente."
+    IDS_DEV_FAILED_START "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os controladores instalados."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:55]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 11]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
     IDS_DEV_LIAR "O ReactOS parou de responder enquanto tentava inicializar este dispositivo e em conseqüência não irá mais tentar inicializá-lo."
     IDS_DEV_NORMAL_CONFLICT "Este dispositivo não pôde encontrar recursos %1 livres para utilizar."
-    IDS_DEV_NOT_VERIFIED "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os drivers instalados."
+    IDS_DEV_NOT_VERIFIED "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os controladores instalados."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:58]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 12]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
     IDS_DEV_NEED_RESTART "Este dispositivo não pode funcionar correctamente até que o computador seja reiniciado."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:59]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 13]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
     IDS_DEV_REGISTRY "O registo pode estar danificado."
     IDS_DEV_WILL_BE_REMOVED "O ReactOS está a remover este dispositivo."
     IDS_DEV_DISABLED "Este dispositivo não foi inicializado."
     IDS_DEV_DISABLED2 "Este dispositivo está desactivado."
-    IDS_DEV_DEVLOADER_NOT_READY "Os carregadores para este dispositivo não puderam carregar os drivers necessários."
+    IDS_DEV_DEVLOADER_NOT_READY "Os carregadores para este dispositivo não puderam carregar os controladores necessários."
     IDS_DEV_DEVLOADER_NOT_READY2 "" /* unused */
     IDS_DEV_DEVLOADER_NOT_READY3 "" /* unused */
-    IDS_DEV_DEVICE_NOT_THERE "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os drivers instalados."
+    IDS_DEV_DEVICE_NOT_THERE "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os controladores instalados."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:71]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 14]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
     IDS_DEV_TOO_EARLY "O ReactOS está no processo de configuração deste dispositivo."
     IDS_DEV_NO_VALID_LOG_CONF "O ReactOS não pode especificar os recursos pra este dispositivo."
-    IDS_DEV_FAILED_INSTALL "Os drivers para este dispositivo não estão instalados."
+    IDS_DEV_FAILED_INSTALL "Os controladores para este dispositivo não estão instalados."
     IDS_DEV_HARDWARE_DISABLED "Este dispositivo está desactivado porque a BIOS para o dispositivo não lhe deu nenhum recurso."
-    IDS_DEV_CANT_SHARE_IRQ "Este dispositivo utiliza um recurso de Requisição de Interrupção (IRQ) que está em uso por outro dispositivo e não pode ser compartilhado.\nDeve alterar a configuração conflitante ou remover o driver de modo real que está causando o conflito."
+    IDS_DEV_CANT_SHARE_IRQ "Este dispositivo utiliza um recurso de Requisição de Interrupção (IRQ) que está em uso por outro dispositivo e não pode ser compartilhado.\nDeve alterar a configuração conflitante ou remover o controlador de modo real que está a causar o conflito."
     IDS_DEV_FAILED_ADD "Este dispositivo não está a funcionar correctamente porque %1 não está a funcionar correctamente."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:78]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 15]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
-    IDS_DEV_DISABLED_SERVICE "O ReactOS não pode instalar os drivers para este dispositivo porque não consegue acessar o drive ou local de rede que possui os arquivos de configuração."
-    IDS_DEV_TRANSLATION_FAILED "Este dispositivo não está a responder ao seu driver."
+    IDS_DEV_DISABLED_SERVICE "O ReactOS não pode instalar os controladores para este dispositivo porque não consegue acessar o controlador ou local de rede que possui os arquivos de configuração."
+    IDS_DEV_TRANSLATION_FAILED "Este dispositivo não está a responder ao seu controlador."
     IDS_DEV_NO_SOFTCONFIG "O ReactOS não pode determinar as configurações para este dispositivo. Consulte a documentação que veio com este dispositivo e utilize a guia Recursos para definir a configuração."
     IDS_DEV_BIOS_TABLE "O firmware de seu sistema não possui informação suficiente para configurar e utilizar este dispositivo correctamente.\nPara utilizar este dispositivo, contacte o fabricante de seu computador para obter uma actualização de firmware ou BIOS."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:82]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 16]
```diff
@@ -40,59 +40,59 @@ STRINGTABLE
...
+    IDS_DEV_FAILED_DRIVER_ENTRY "O ReactOS não pode inicializar o controlador do dispositivo para este hardware."
+    IDS_DEV_DRIVER_FAILED_PRIOR_UNLOAD "O ReactOS não pode carregar o controlador do dispositivo para este hardware porque uma instância anterior do controlador do dispositivo ainda está na memória."
+    IDS_DEV_DRIVER_FAILED_LOAD "O ReactOS não pode carregar o controlador do dispositivo para este hardware. O controlador pode estar corrompido ou ausente."
     IDS_DEV_DRIVER_SERVICE_KEY_INVALID "O ReactOS não pode acessar este hardware porque suas informações sobre a chave do serviço não estão no Registo ou estão registadas incorrectamente."
-    IDS_DEV_LEGACY_SERVICE_NO_DEVICES "O ReactOS carregou o driver do dispositivo para este hardware correctamente mas não consegue encontrar o dispositivo de hardware."
-    IDS_DEV_DUPLICATE_DEVICE "O ReactOS não pode carregar o driver do dispositivo para este hardware porque já existe uma instância deste dispositivo em execução no sistema."
+    IDS_DEV_LEGACY_SERVICE_NO_DEVICES "O ReactOS carregou o controlador do dispositivo para este hardware correctamente mas não consegue encontrar o dispositivo de hardware."
```
Comment (HBelusca) [inline dll/win32/devmgr/lang/pt-PT.rc:88]: "corretamente" ?

File: dll/win32/netid/lang/pt-PT.rc [hunk 17]
```diff
@@ -71,20 +71,20 @@ BEGIN
     4 "Alterar o nome do computador"
     5 "Grupo de trabalho:"
     6 "Domínio:"
-    7 "The domain name ""%1"" does not conform to Internet Domain Name Service specifications, although it is a legal ReactOS name. You must use ReactOS DNS server for non-standard names."
-    8 "The domain name ""%1"" is not properly formatted. Periods (.) are used to separate domains. Each domain is limited to 63 characters. Example: domain-1.reactos.org."
-    10 "The new computer name ""%1"" is too long. The name may not be longer than 63 characters."
-    11 "The new computer name entered is not properly formatted. Standard names may contain letters (a-z, A-Z), numbers (0-9), and hyphens (-), but no spaces or periods (.). The name may not consist entirely of digits."
+    7 "O nome de domínio ""%1"" não está em conformidade com as especificações do Serviço de Nomes de Domínio da Internet, embora seja um nome ReactOS legal. Deve usar o servidor DNS ReactOS para nomes não padrão."
+    8 "O nome de domínio ""%1"" não está formatado correctamente. Pontos (.) são usados para separar domínios. Cada domínio é limitado a 63 caracteres. Exemplo: domínio-1.reactos.org."
```
Comment (HBelusca) [inline dll/win32/netid/lang/pt-PT.rc:75]: "corretamente" ?

File: dll/win32/netid/lang/pt-PT.rc [hunk 18]
```diff
@@ -71,20 +71,20 @@ BEGIN
...
+    10 "O novo nome do computador ""%1"" é muito longo. O nome não pode ter mais de 63 caracteres."
+    11 "O novo nome de computador inserido não está formatado correctamente. Os nomes padrão podem conter letras (az, AZ), números (0-9) e hifens (-), mas sem espaços ou pontos (.). O nome não pode consistir inteiramente de dígitos."
```
Comment (HBelusca) [inline dll/win32/netid/lang/pt-PT.rc:77]: "corretamente" ?

File: dll/win32/devmgr/lang/pt-PT.rc (hunk 2 above)
Comment (zecarlos1957) [reply dll/win32/devmgr/lang/pt-PT.rc:53]: Incorretamente is PT Brasilian, incorrectamente is PT  Portugal

File: dll/win32/devmgr/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply dll/win32/devmgr/lang/pt-PT.rc:87]: incorreCtamente is Portuguese from Portugal

**Comment by HBelusca** [comment]
Thanks @zecarlos1957 ! The online dictionary wasn't clear about the portuguese brazil fact.

**Pull request merged**